sawmill_api/lib/oltp.py

```diff
@@ -116,3 +136,55 @@ def example(cursor):
     )
     cursor.execute(sql)
     return rows_to_dicts(cursor)
+
+
+@with_cursor()
+def get_api_settings(cursor) -> dict[str, list]:
+    """
+    Obtain the settings used by the API.
+    """
+    sql = dedent(
+        """\
+        SELECT
+            section,
+            name,
+            value
+        FROM api.settings
+        """
+    )
+    cursor.execute(sql)
+    grouped_by_section = {}
+    for row in cursor.fetchall():
+        section, name, value = row
+        section_group = grouped_by_section.setdefault(section, {})
+        section_group[name] = value
+    return grouped_by_section
+
+
+@with_cursor()
+def set_api_setting(cursor, setting: settings.Section):
+    """
+    Save the current section settings ot the database.
+    """
+    sql = dedent(
+        """\
+        INSERT INTO api.settings (section, name, value)
```

Comment:
Cockroach doesn't support name aliases at this time. So no `INSERT INTO some_table (a,b,c) AS st`, where I can lazily type out `st` instead of the _full name_. NBD.